pending/ashampoo_burning_studio.xml

```diff
@@ -0,0 +1,241 @@
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2018\VIDEO_TS Disc Project\SelectVideoTSFolder" name="VideoTSPath"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Audio Disc Project\SaveDialog_CPlaylistDlgEx" name="InitialDirectory"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Backup Project\BackupOptions" name="CustomLocation"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Browse Image Project\BrowseImageFile" name="ImagePath"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Browse Image Project\SaveDialog_SelectImageBrowse" name="InitialDirectory"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Burn Image Project\SaveDialog_SelectImageBrowse" name="InitialDirectory"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Burn Image Project\SelectImage" name="ImagePath"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Data Disc Project\DumpImage" name="ImagePath"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Data Disc Project\SaveDialog_AddFilesAndDirs" name="InitialDirectory"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Logs"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\tempFiles"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\Unknown Project"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\VCD Project\SaveDialog_OnAddMovies" name="InitialDirectory"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\VIDEO_TS Disc Project\DumpImage" name="ImagePath"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 2019\VIDEO_TS Disc Project\SelectVideoTSFolder" name="VideoTSPath"/>
+  <!-- And was:
+  </option>
+  <option id="images">
+    <label>Last Images</label>
+    <description>Delete the data</description>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 6\Burn Image Project\SelectImage" name="ImagePath"/>
+    <action command="winreg" path="HKCU\Software\Ashampoo\Ashampoo Burning Studio 6\Dump Image Project\DumpImage" name="ImagePath"/>
+  -->
+  </option>
+  <option id="logs">
+    <label>Logs</label>
+    <description>Delete the logs</description>
+    <action command="delete" search="walk.all" path="%AppData%\Ashampoo\Ashampoo Burning Studio *\Logs\"/>
+    <!-- Delete *.log recursive: -->
+    <action command="delete" search="walk.files" path="%AppData%\Ashampoo\" regex="\.[Ll][Oo][Gg]$"/>
+  </option>
+</cleaner>
+<!--
+Additionaly WinApp2.ini have this rules...
```

Comment:
I did not compare this to the rules in Winapp2.ini, but here is a general comment.  If the Winapp2.ini cleaner works or can be made to work, it's best not to translate it manually into a CleanerML file. One problem is each will have to be maintained, like when  the application changes or new files are discovered.

A good reason to start a CleanerML file is if it is impossible for the Winapp2.ini cleaner to do a good job. For example, Winapp2.ini can't edit .ini files, edit json, do anything on Linux, etc.

Reply:
I have some reasons why I give the knowledge of my "Reverse Engineering" to the CleanerML Project and also think the WinApp2.ini data should be ported to CleanerML...
One reason is my experience with try to volunteering by WinApp2.ini: It was hard to get my changes in and they gave me the feeling, that they think they are "special" - and I don't want to fight for volunteering anymore!
Also: The data is now used by many other companies then Periform - e.g. Avira - and I don't like this!
Then: If we "convert" the data to CleanerML, we can split it into better categories like "logs", "temp", etc. what gives the user more control...
A other reason: IMHO users should be able to use BleachBit without WinApp2.ini!
We should have a "WinApp2.ini trimmer" for help to convert data to CleanerML, but I will fill a issue about that in the wishlist, later...